cos-fleetshard-operator-debezium-it/src/test/resources/DebeziumConnectorReifySchemalessJson.feature

```diff
@@ -39,6 +39,7 @@ Feature: Debezium Connector Reify
      And the kc has an entry at path "$.spec.authentication.passwordSecret.secretName" with value "${cos.managed.connector.name}-config"
      And the kc has an entry at path "$.spec.authentication.passwordSecret.password" with value "_kafka.client.secret"
      And the kc has an entry at path "$.spec.image" with value "quay.io/rhoas/cos-connector-debezium-postgres@sha256:b67d0ef4d4638bd5b6e71e2ccc30d5f7d5f74738db94dae53504077de7df5cff"
+     And the kc has an entry at path "$.spec.imagePullSecrets[0].name" with value "cos-pull-secret"
```

Comment:
This needs to be copied to the other test cases for AVRO and SchemalessJson too.

Reply:
And I think the coordinates still fail. Like in the unit tests it should be something like `$.spec.template.pod.imagePullSecrets[0].name`, when I'm not mistaken?

Reply:
```suggestion
     And the kc has an entry at path "$.spec.template.pod.imagePullSecrets[0].name" with value "cos-pull-secret"
```